src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -20,9 +20,6 @@ trait Ledger {
 
 class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with Logger {
 
-  case class BlockResult(worldState: InMemoryWorldStateProxy, gasUsed: BigInt = 0, receipts: Seq[Receipt] = Nil)
```

Comment:
Nvm, for me it's ok to leave it as it is because the other PR is going to happen soon